### PR TITLE
feat(cli-repl): allow loading files from command line MONGOSH-326

### PR DESCRIPTION
**Pull request by addaleax**

### CODE REVIEW

**Comment by addaleax** [comment]
I’ve tacked on `--eval` support to this PR, as it’s very related and it probably makes sense to review them as one.

File: packages/cli-repl/src/cli-repl.ts [hunk 1]
```diff
@@ -151,10 +151,41 @@ class CliRepl {
...
     await this.loadRcFiles();
     await this.mongoshRepl.startRepl(initialized);
   }
 
+  listCommandLineLoadFiles(): string[] {
+    const startIndex = this.cliOptions.nodb ? 0 : 1;
+    return (this.cliOptions._ ?? []).slice(startIndex);
+  }
+
+  async loadCommandLineFiles(files: string[]) {
```
Comment (rose-m) [inline packages/cli-repl/src/cli-repl.ts:171]: Nit: could we rename that to `loadCommandLineFilesAndEval`? Just from reading it seems more like a "side effect" that it also loads the eval code.

**Review by rose-m — APPROVED** [review]
Just a minor method name thing :)

File: packages/cli-repl/src/cli-repl.ts (hunk 1 above)
Comment (addaleax) [reply packages/cli-repl/src/cli-repl.ts:171]: ah, sure :) will do that along with fixing the test that’s still borked up (which is due to nyc/coverage, it seems…)

**Review by rose-m — APPROVED** [review]

**Pull request merged**